modules/ROOT/pages/install-checklist.adoc

```diff
@@ -17,6 +17,8 @@ All prerequisites apply to all xref:prereq-platform#supported-environments[Suppo
 ** Working with Terraform. This is only required if your environment includes AWS.
 ** Remote connection mechanisms, such as SSL and SSH.
 
+* Contact MuleSoft professional services prior to installation for assistance.
```

Comment:
how do they contact professional services?
Are we saying they won't get through the install without contacting PS, or that *if* they need assistance, they must contact PS? It's ambiguous here.

Reply:
They **must** contact PS. It's not supported for customers to install neither upgrade on their own. This is not a self-service product. I'd recommend a more bold statement on this, the current wording is also ambiguous to me.